_aggregations/bucket/children.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: default
+title: Children
+parent: Bucket aggregations
+grand_parent: Aggregations
+nav_order: 15
+---
+
+# Children
+
+The `children` aggregation connects parent documents with their related child documents. This allows you to analyze relationships between different types of data in a single query, rather than having to run multiple queries and combine the results yourself.
```

Comment:
```suggestion
The `children` aggregation connects parent documents with their related child documents. This allows you to analyze relationships between different types of data in a single query, rather than needing to run multiple queries and combine the results manually.
```